ci/do_ci.ps1

```diff
@@ -40,7 +40,7 @@ switch ($action) {
     if ($exit -ne 0) {
       exit $exit
     }
-    cmake --build .
+    cmake --build . -j
```

Comment:
Just curious, Will CMake automatically use Ninja as build system, or we need to configure it explicitly?
The build time is reduced from 14 min to 10 min for Windows, so there seems to be some level of  parallelism.

Reply:
With my solution (getting number of physical cores from Powershell command), I assume that it takes less time than before, what do you think?

Reply:
Yeah, sorry for inconvenience. After discussion on issue, I decided to use Ninja but it does not work at all. So, I will update the PR title and do you want me to change commit message either?